src/components/Breadcrumb/index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable import/no-extraneous-dependencies,no-script-url */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Here we need to use internals icon for the component since fontawesome is not a library dependency. Ask for icons to @TahimiLeonBravo.